noir/noir-repo/aztec_macros/src/transforms/note_interface.rs

```diff
@@ -209,6 +298,7 @@ fn generate_note_get_header(
 
     let (function_ast, errors) = parse_program(&function_source);
     if !errors.is_empty() {
+        dbg!(errors);
```

Comment:
```suggestion
```

Reply:
This is something we are doing in every parsing of autogenerated functions via string templating and was missing from this particular function. It has proven to be useful.

Reply:
Ah ok, it looked like something left in from development.